pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -758,8 +778,9 @@ impl Timeline {
             return Err(CompactionError::ShuttingDown);
         }
 
-        let keyspace = self.collect_keyspace(end_lsn, ctx).await?;
-        let mut adaptor = TimelineAdaptor::new(self, (end_lsn, keyspace));
+        let (dense_ks, _sparse_ks) = self.collect_keyspace(end_lsn, ctx).await?;
+        // ignore sparse_keyspace for now, compact it in the future.
```

Comment:
can you turn this into a `TODO`?

Reply:
fixed in https://github.com/neondatabase/neon/pull/7503/commits/804ea39e084fc70596a56ad141380dd92a453b0c